bookkeeper-proto/src/main/proto/DataFormats.proto

```diff
@@ -60,6 +60,8 @@ message LedgerMetadataFormat {
         optional bytes value = 2;
     }
     repeated cMetadataMapEntry customMetadata = 11;
+
+    optional int64 creatorId = 12;
```

Comment:
we need to a flag to control whether to add `creatorId` or not. since we are still using text format for storing ledger metadata.

Reply:
This patch only added it to Version 3 which afaict is using the normal protobuf format, so old clients should ignore it.  Is that right?

Reply:
@athanatos 

sorry I misread that. you are right.

Reply:
@sijie ^ ?

Reply:
I thought I have approved the pull request since you have answered my question :)

Reply:
Oh, I think I added my comment without reloading the page.  Sorry!